### PR TITLE
add support for Power IPSO

### PR DESCRIPTION
**Pull request by thetoster**
Signed-off-by: bartlomiej.zarnowski <bartlomiej.zarnowski@imgtec.com>

### CODE REVIEW

File: lwm2m-client-ipso-power.h [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+#define IPSO_POWER_MAX_RANGE_VALUE                    5604
+#define IPSO_POWER_RESET_MIN_AND_MAX_MEASURED_VALUES  5605
+#define IPSO_POWER_APPLICATION_TYPE                   5750
+
+/***************************************************************************************************
+ * Functions
+ **************************************************************************************************/
+
+/**
+ * @brief Define single IPSO Temperature object (with ID 3328) and all resources bound to it. It also takes min and max measurable powers as a parameters.
```
Comment (manohar-narkhede-imgtec) [inline lwm2m-client-ipso-power.h]: Should be Power

**Comment by thetoster** [comment]
Fixed Temperature typo. About Application type I'm against this change. Object informs about what it measures or to what this can be applied, not platform on which it's deployed. Moreover this can be customised by enduser as he needs.

**Comment by manohar-narkhede-imgtec** [comment]
Actually the Application type should specify "The application type of the sensor or actuator as a string depending on the use case". I think if the power of a particular device e.g. Light is getting measured then, that should be mentioned as application type. In this case as the power of Clicker is getting measured, the application type can be "Clicker".

File: lwm2m-client-ipso-power.c [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+    char ApplicatioType[APPLICATION_TYPE_MAX_LEN];
+} IPSOPower;
+
+/***************************************************************************************************
+ * Globals
+ **************************************************************************************************/
+
+static IPSOPower PowerStorage = {
+  .Units = "Watt",
+  .ApplicatioType = "power"
```
Comment (abhijit-mahajani-imgtec) [inline lwm2m-client-ipso-power.c]: Fix the typo - ApplicatioType

**Comment by thetoster** [comment]
done.

**Comment by manohar-narkhede-imgtec** [comment]
Approved.

**Pull request merged**